Arthas' Event Chain

zumbak04 opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- When the Plague of Lordaeron starts, the king of the targeted kingdom (Terenas) may send his trustworthy commander (Arthas) to investigate the outbreak. Arthas may invite a player if Arthas has more than 0 opinion of them. The main characters are:
  - Arthas
  - Terenas
  - Player (generic Terenas' vassal)
- When you're in the Arthas' expedition you wear armor.

# Design Document
#777

# Tasks:
- [x] Set intervals between events
  - Check WC3 missions for dates
  - [x] Adjust history to fit it
- [x] Change activity icon
- [x] Change scenes of plague events to new ones
- [x] Add Sadistic to Stratholme
- [x] Replace liege by leader
- [x] Arthas wears hood in the last events
- [x] Stop event chain if Ner'zhul dies
- [x] Add an alternative description if Hearthglen wasn't found
- [x] Finish `on_trigger_fail` if Stratholme wasn't found
- [x] Add Lordaeron throne room
- [x] Arthas shouldn't wear a hood after he becomes ruler
- [x] Fix no ship event
- [x] Arthas stays king and holds the capital

<!--
Before the merge, we recommend you to do these tests with your branch.
-->
## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
@Warcraft-GoA-Development-Team/ck3-testers play Terenas, Arthas or generic Terenas' vassal and try different options in this event chain.

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/files?authenticity_token=HdCBWFbgecjuZQBS%2FFmFv%2Fjci4F9SxwA%2FBzNkYAM%2BnVZ3ECtgHlsfJ1595UkOcfdTtKLFr%2BkoNBFoKu5vbStXg%3D%3D&file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here's the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).

ElMariuso reviewed (commented):
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46578034/148776930-8e0fef43-eb37-48fe-b991-d4346e58de6d.png)

</details>

I think it would be more impactful if we had the official sentences in the options.
Like:
- This entire city must be purged.
- We must evacuate the inhabitants!
- We need to quarantine this zone.

zumbak04 commented:
> I think it would be more impactful if we had the official sentences in the options. Like:
> 
> * This entire city must be purged.
> * We must evacuate the inhabitants!
> * We need to quarantine this zone.

They're too long and won't fit in options.

> Also, it can be cool to have Stratholme background.

May do.

Soltysutine reviewed (approved):
**Approved**

- Issue with the weird event update if you were to play as Terenas when he dies has been fixed
- When playing in any position of the High Kingdom of Lordaeron (Arthas, Terenas, or the vassals) events play out like they should for the respective position you play in.

Furthermore, I can say this can be merged with no problems present at the current moment

pedromdpereira commented:
I played this version for a while and the story is on point. I would make a quick suggestion. When Arthas picks up frostmourne, his prowess should improve a little. If you agree, I tested it by adding this statement to wc_arthas_story.70:
`add_prowess_skill = 5`

Other than that it is working correctly.

zumbak04 commented:
> I played this version for a while and the story is on point. I would make a quick suggestion. When Arthas picks up frostmourne, his prowess should improve a little. If you agree, I tested it by adding this statement to wc_arthas_story.70: `add_prowess_skill = 5`

We plan to add Frostmourne with the next DLC.

arithon commented on localization/english/wc_scripted_modifiers_l_english.yml:
```diff
@@ -0,0 +1,2 @@
+﻿l_english:
+ RELIGION_OBEY_SCOURGE:0 "Obeys the [scourge|E]: $VALUE|=+0$"
```
Why not put this file into the modifier folder?

zumbak04 replied:
> Why not put this file into the modifier folder?

Which one?

arithon replied:
the only one with _modifier_ in its name... 😅

zumbak04 replied:
> the only one with _modifier_ in its name... 😅

It's file for modifiers, not scripted modifiers.

Gaben1025 commented:
Should "you" be capitalized?
![image](https://user-images.githubusercontent.com/74790101/149040607-5b639345-0962-474e-9e35-e2df55fb93ce.png)

TheDrPibb1999 reviewed (changes requested):
Without rehashing some of the past reviews, here are a few of my observations

#1
![Capture](https://user-images.githubusercontent.com/79172797/149250576-0a0e396b-f5d5-4320-aab8-a63aaf98ea15.JPG)

I have circled the area of my concern, with this being a statement listing only two things, I feel a comma is not needed, and in my opinion, having it be "Muradin's training has made me a capable warrior and Uther has shown me the ways of the light" .....  Source: Grammarly - **"But you don't need a comma when you're simply listing two items "**

#2
![Capture 2](https://user-images.githubusercontent.com/79172797/149250975-f5453b5e-7cee-4466-bfb2-26b097f4b9ff.JPG)

 I have two areas circled here. In the blue, I find this sentence kinda awkward - a few alternates I can think of are "Local reports describe the dead being resurrected into something inhuman, creating monstrosities." or maybe just some simple reformating - but a slight observation nonetheless... In the black I have the same idea, having it be two separate sentences makes it appear to me as them being two incomplete sentences, joining them with a comma could be a simple fix, or something along the lines of "Trouble is brewing in the north, and I must act quickly"

#3 
![Capture 3](https://user-images.githubusercontent.com/79172797/149251476-161328a3-0852-4434-bf5a-6ce511f9e1d7.JPG)

This isn't a complaint, I just want to say this looks fricking amazing!!!!

#4
![Capture 4](https://user-images.githubusercontent.com/79172797/149251789-ee28b0f7-4c32-49fc-8bb5-07aff43def62.JPG)

A very slight note, to me, the "are" is not needed. Having it simply removed could be a solution or maybe changing it to "The men become rattled as they discover the abominations and armed skeletons encircling our expedition" 

#5
![cap 5](https://user-images.githubusercontent.com/79172797/149252077-1c889a66-9243-4358-964d-2a1c7c596a0a.JPG)

Very slight thing, the term "wizard" seems unfit for this scenario, having it be mage (as in the localization they are referred to as robed mages) or even necromancer could be better. 

#6 
![cap 6](https://user-images.githubusercontent.com/79172797/149258138-f80f284a-1dc3-44de-9027-c92389b04528.JPG)

Circled in red, I feel this is a bit weird and could be rephrased better, such as "The necromancer's words linger in the back in my mind"... circled in blue, I feel this sentence is kinda quick and incomplete, maybe something such as "As we traverse farther into the city, I make note of the soldiers training in the village square" 

#7 
![cap 67](https://user-images.githubusercontent.com/79172797/149258352-59230d29-2fc7-4191-9e9b-fe679964d15a.JPG)

Circled in black, I feel these two can be combined, and maybe some more flavor could be added, such as " While the soldiers begin to march, I noticed the shipments of grains and immediately become alarmed. The crates bear the sigil of Andorharl!"

#8 
![Capture99](https://user-images.githubusercontent.com/79172797/149258939-64371902-6b6a-4e88-b04f-d6ab743ac5e2.JPG)

I feel this sentence could be rephrased better, such as "When my forces reach the shores of this barren land, almost all of the men become unsettled by the dark shroud lurking above our fleet, along with the sharp wind in the air. I command our forces to swiftly make way inland."

#9
![Inkedcap cap_LI](https://user-images.githubusercontent.com/79172797/149259582-83700476-9024-47bc-90e9-91892128646f.jpg)

I think this can be rephrased as well, it seems a bit short for such a crucial moment, possibly can be changed to something along the lines of "After a short exchange of combat, I achieve the upper hand and drive Frostmourne into the dreadlord, as I pull out the runeblade, Mal'Ganis drawls his final breath. With the corpse of the demon slowly disintegrating, the voice of the etc etc etc"

#10 
![Capture7](https://user-images.githubusercontent.com/79172797/149259800-98a2f20c-78ad-4a76-b292-67d040910ee7.JPG)
![Capture](https://user-images.githubusercontent.com/79172797/149259963-961b5f6f-2c3a-4bc5-83c8-8234007f785c.JPG)

Weird spaces.

#11 
![Capture](https://user-images.githubusercontent.com/79172797/149260036-d5b10b8e-7b6a-49e6-b5f0-8dbe26325870.JPG)

Personally I think this is absolutely amazing... 



Thats about all my notes, I went through the checklist and everything I saw worked perfectly (expect for the previously mentioned stuff) amazing!!!!

TheDrPibb1999 commented:
One more note I forgot to add. When playing as Terenas, Arthas does not have his hood on when he slays you. 
![Capture](https://user-images.githubusercontent.com/79172797/149260850-f2be810d-d3da-4e1d-aa88-8b9f8beff58e.JPG)

TheDrPibb1999 commented:
> A very slight note, to me, the "are" is not needed. Having it simply removed could be a solution or maybe changing it to "The men become rattled as they discover the abominations and armed skeletons encircling our expedition"

I was referring to the second "are" not the first

TheDrPibb1999 commented:
I can also confirm that when playing as a vassal Arthas does not have his hood either
![cap](https://user-images.githubusercontent.com/79172797/149274838-b8a28b2a-7bb0-41c9-aa36-16da2b986748.JPG)

zumbak04 commented:
@TheDrPibb1999 @Gaben1025 please make localization changes like it's explained here:

> @Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/files?authenticity_token=HdCBWFbgecjuZQBS%2FFmFv%2Fjci4F9SxwA%2FBzNkYAM%2BnVZ3ECtgHlsfJ1595UkOcfdTtKLFr%2BkoNBFoKu5vbStXg%3D%3D&file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here's the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).

zumbak04 commented:
> Should "you" be capitalized?

Done in 03ae91b53e58d82877665205a269751c59ce0255.

> I'm not sure if this was on purpose, but Arthas's expedition is still in Andorhal with Arthas attending, though according to events Arthas is in Northrend already

It's on purpose. The activity location can't be moved.

> 1. When using a ";" the stuff coming before and after it should be directly connected. I would replace it with a "..." in the first paragraph.
> 2. I would remove the "," in the second paragraph right after "Dwarven camp" because it creates an awkward pause
> 3. The first sentence in the third paragraph has a typo after the comma, idk how to fix since idk what you meant

Fixed in 1bba1874184c3b92cfd905fee608f82134403564.

> I would change to "Having escaped Mal'Ganis, we defeated the guardian of the cave, now nothing will stop Muradin and I from reaching Frostmourne."

Changed in 1bba1874184c3b92cfd905fee608f82134403564.

> Arthas took Lorderan but is still at war with the scourge and allied to Stormwind,Kul Tiras, etc.

On the screenshot, you play Arthas. As a player, you may do anything you want. The AI Arthas would surrender.

zumbak04 commented:
> Very slight thing, the term "wizard" seems unfit for this scenario, having it be mage (as in the localization they are referred to as robed mages) or even necromancer could be better.

It's a quote from the Warcraft 3.

zumbak04 commented:
> One more note I forgot to add. When playing as Terenas, Arthas does not have his hood on when he slays you. ![Capture](https://user-images.githubusercontent.com/79172797/149260850-f2be810d-d3da-4e1d-aa88-8b9f8beff58e.JPG)

Changed in 032214338c4177b49db901e5d2de2ecdf9713b94.

TheGroxEmpire reviewed (commented):
- After finishing the questline and usurping Lordaeron titles as Arthas, the player is replacing Terenas and fighting the war as it was against the Lich King, the player should have automatically surrendered after usurping Lordaeron:
<details>

![image](https://user-images.githubusercontent.com/47816228/149339866-cb2bf2e4-8c31-4f7f-83ee-cd29ed543506.png)
</details>

- There should be a special panel that indicates where I could open the Outbreak Expedition page; something like a Great Holy War badge that is obvious to the player. I'm clueless on how I could open the Outbreak Expedition panel. The Icon indicator on top of the expedition's current location didn't show up 99% of the time and is not obvious enough for players.

- This option should be clarified further on how it would impact the expedition. "To the ends of the earth!" should have something along the line: you pursue Mal'Ganis in the frigid land of Northrend.
<details>

![image](https://user-images.githubusercontent.com/47816228/149356488-077a3445-b4bd-461c-9a15-ec63e7cf1646.png)
</details>

- Playing as the random generic Vassal. These events are written as Arthas PoV and are incompatible for generic Vassal player:
<details>

![image](https://user-images.githubusercontent.com/47816228/149352382-c15f6c02-2efa-4f42-bca5-e351332b2c17.png)

![image](https://user-images.githubusercontent.com/47816228/149353092-1b66d376-77fd-4954-83e7-fb61ad215f42.png)

</details>

- If Terenas died during the event, Arthas would get an event to recall him back:
<details>

![image](https://user-images.githubusercontent.com/47816228/149355391-3d363e6c-40a1-4e3d-b265-b2811bf9813b.png)

</details>

TheDrPibb1999 reviewed (changes requested):
I do not know the exact duration the event last long for lore wise in regards to Arthas going to Northrend and returning back home to slay Terenes, however, I wanted to test to see how far I could get my fleet (originating in Strathholme) to Northrend and back... I conducted this experiment the second Arthas swears to follow Mal'ganis to ends of the Earth. While I understand the variation between to games naval speed and lore-wise, my fleet was only halfway to Northrend for the first time when Arthas was already plunging his blade into his father. I feel like the events should be spaced a bit more realistically at least for the standards of the game.
<details>
<summary>Click to expand</summary>

![Capture](https://user-images.githubusercontent.com/79172797/149359610-381c4598-fb42-4ce2-8180-ef79916dfb46.JPG)
</details>

I just noticed this after I posted this review so Im just gonna edit it... 
Whenever you play as anyone other then Arthas and he usurps Lorderon, he marries some random naked woman lmfao, first time it was Jania which made me laugh, but I doubt Jania would want to marry an Undead Arthas and agree to be naked 24/7, even if Arthas dreamed about it... This only happens if you are a vassal or Terenas. Arthas becomes married to the said naked woman the SECOND he become high king (ai arthas surrendered and that is why he is a duke there in the first pic)
<details>
<summary>Click to expand</summary>

![Capture](https://user-images.githubusercontent.com/79172797/149361273-3eb8f0f7-b244-4c3b-85c8-723d7b71aef6.JPG)
</details>
<details>
<summary>Click to expand</summary>
![Captures](https://user-images.githubusercontent.com/79172797/149361283-1a303dd9-1dc0-46e9-8319-144e2b7c6677.JPG)
</details>
![Captures](https://user-images.githubusercontent.com/79172797/149361283-1a303dd9-1dc0-46e9-8319-144e2b7c6677.JPG)

This happened on multiple occasions
 
Next, I noticed that sometimes that it isn't Murdain in Northrend but some random other dwarf, is that intended sometimes??
<details>
<summary>Click to expand</summary>
![Capture](https://user-images.githubusercontent.com/79172797/149364184-8bed7311-6d2e-49f8-bfc5-6693ff8ae082.JPG)
</details>

Lastly, I also noticed that is a random chance if Mal'ganis is either wearing armor or normal attire when getting slain by Arthas 
<details>
<summary>Click to expand</summary>
![Capturesss](https://user-images.githubusercontent.com/79172797/149364283-bddcc584-ce54-4bcd-9999-588a60aaa69b.JPG)
</details>
<details>
<summary>Click to expand</summary>
![Capture](https://user-images.githubusercontent.com/79172797/149364511-7d1c5f77-00ee-4806-9f80-50242fd4d282.JPG)
</details>

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_55_DESC_arthas_ending:0 "After establishing our base camp, [muradin.GetSheHe] said that [muradin.GetHerHis] purpose here is to recover a runeblade called Frostmourne. Could it be a similar blade like the one that has haunted my dreams?"
+ wc_arthas_story_55_DESC_group_beggining:0 "After a month of sailing, we’ve arrived in Northrend. The howling wind cuts to the bone.\n\nWe didn’t travel far inland until we found a [muradin.GetCulture.GetGroup.GetName] camp and their leader [muradin.GetFirstName] [muradin.GetDynastyHouseName]"
+ wc_arthas_story_55_DESC_group_muradin:0 ", [arthas.GetTitledFirstNamePossessive] old teacher and friend"
+ wc_arthas_story_55_DESC_group_ending:0 ". [muradin.GetSheHe|U] says that [muradin.GetHerHis] purpose here is to recover a runeblade called Frostmourne.\n\nWe joined up with [muradin.GetFirstNamePossessive] encampment and are now prepared to battle the frozen undead that roam these lands. Some of [muradin.GetFirstNamePossessive] forces have been cut off from [muradin.GetHerHis] camp. Our expediti ...
```
This is as an ending of the previous line (which could be about being Arthas' teacher or not.

arithon replied:
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

zumbak04 replied:
It's not a typo and you shouldn't fix it.

zumbak04 replied:
It's not a typo and you shouldn't fix it.

arithon reviewed (approved):
english loc should be fine (LGTM), german needs some CustomLoc rework, but that shouldn't stop you from merging

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_55_DESC_arthas_not_muradin:0 "[muradin.GetFirstName] [muradin.GetDynastyHouseName].\n\n"
+ wc_arthas_story_55_DESC_arthas_ending:0 "After establishing our base camp, [muradin.GetSheHe] said that [muradin.GetHerHis] purpose here is to recover a runeblade called Frostmourne. Could it be a similar blade like the one that has haunted my dreams?"
+ wc_arthas_story_55_DESC_group_beggining:0 "After a month of sailing, we’ve arrived in Northrend. The howling wind cuts to the bone.\n\nWe didn’t travel far inland until we found a [muradin.GetCulture.GetGroup.GetName] camp and their leader [muradin.GetFirstName] [muradin.GetDynastyHouseName]"
+ wc_arthas_story_55_DESC_group_muradin:0 ", [arthas.GetTitledFirstNamePossessive] old teacher and friend"
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

zumbak04 replied:
It's not a typo and you shouldn't fix it.

zumbak04 replied:
It's not a typo and you shouldn't fix it.

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_55_DESC_arthas_muradin:0 "[muradin.GetFirstName] [muradin.GetDynastyHouseName], my old mentor and friend.\n\n"
+ wc_arthas_story_55_DESC_arthas_not_muradin:0 "[muradin.GetFirstName] [muradin.GetDynastyHouseName].\n\n"
+ wc_arthas_story_55_DESC_arthas_ending:0 "After establishing our base camp, [muradin.GetSheHe] said that [muradin.GetHerHis] purpose here is to recover a runeblade called Frostmourne. Could it be a similar blade like the one that has haunted my dreams?"
+ wc_arthas_story_55_DESC_group_beggining:0 "After a month of sailing, we’ve arrived in Northrend. The howling wind cuts to the bone.\n\nWe didn’t travel far inland until we found a [muradin.GetCulture.GetGroup.GetName] camp and their leader [muradin.GetFirstName] [muradin.GetDynastyHouseName]"
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

zumbak04 replied:
It's not a typo and you shouldn't fix it.

zumbak04 replied:
It's not a typo and you shouldn't fix it.

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_75_OPT_not_arthas:0 "[arthas.GetTitledFirstName], where are you?!"
+ 
+ wc_arthas_story_80_TITLE:0 "[arthas.GetFirstNamePossessive] Betrayal"
+ wc_arthas_story_80_DESC:0 ""[arthas.GetFirstName]! You are back!"\n\n[arthas.GetTitledFirstName], or what is left of [arthas.GetHerHim], looks as dead as the things we fought. I grab the nearest weapon, only to see that the undead swarm the camp and catch all of us by surprise.\n\nSomething slices my back, causing me to fall on the ground. My friends, all falling one by one as they shout in pain ...
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_65_OPT_mercenaries:0 "In this land we will stand or fall together."
+ 
+ wc_arthas_story_70_TITLE:0 "Frostmourne"
+ wc_arthas_story_70_DESC:0 "Having escaped [malganis.GetFirstName], we defeated the guardian of the cave, now nothing will stop [muradin.GetFirstName] and I from reaching Frostmourne now.\n\n[muradin.GetFirstName] kneels to read the plague on the altar, "Hold, lad. There's an inscription on the dais. It's a warning. It says, "Whomsoever takes up this blade shall wield power eternal. Just as the b ...
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_35_OPT_B_evacuate:0 "Enough... I'm leaving."
+ 
+ wc_arthas_story_40_DESC_purge_group:0 "Following [arthas.GetTitledFirstNamePossessive] decision, torment screams and fierce battles echo through the streets; the boundary between living and undead, good and evil, blur undistinguishable. The Culling is unbroken during the day and continues into the night by the flames of the burning city. Our spirits are held up solely by [arthas.GetTitledFirstNa ...
+ wc_arthas_story_40_DESC_purge_world:0 "Troubling news arrives regarding from the [lordaeron.GetName].\n\nThe city of [stratholme.GetNameNoTier] has been purged by its own people. The current Plague of Undeath that has gripped [lordaeron.GetNameNoTier] spread to the city.\n\nWhen [arthas.GetTitledFirstNamePossessive] forces arrived outside the walls of the city, the first thing they did was inspe ...
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ 
+ wc_arthas_story_40_DESC_purge_group:0 "Following [arthas.GetTitledFirstNamePossessive] decision, torment screams and fierce battles echo through the streets; the boundary between living and undead, good and evil, blur undistinguishable. The Culling is unbroken during the day and continues into the night by the flames of the burning city. Our spirits are held up solely by [arthas.GetTitledFirstNa ...
+ wc_arthas_story_40_DESC_purge_world:0 "Troubling news arrives regarding from the [lordaeron.GetName].\n\nThe city of [stratholme.GetNameNoTier] has been purged by its own people. The current Plague of Undeath that has gripped [lordaeron.GetNameNoTier] spread to the city.\n\nWhen [arthas.GetTitledFirstNamePossessive] forces arrived outside the walls of the city, the first thing they did was inspe ...
+ wc_arthas_story_40_DESC_evacuate_group:0 "The days pass by with just small fights breaking out through the city.\n\nAs the evacuation process continued in the outskirts, our soldiers searched the plagued city of [stratholme.GetNameNoTier] for the undead lord. We encountered [malganis.GetFirstName] and were victorious in defeating [malganis.GetHerHis] forces, but the undead lord managed to flee t ...
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_60_OPT_run:0 "To hell with the undead!"
+ 
+ wc_arthas_story_65_DESC_arthas:0 "If my warriors abandon me, I'll never defeat [malganis.GetFirstName]. The ships must be burned before the men reach the shore. We'll need more men to fight our way through the undead. [muradin.GetTitledFirstName] mentions a few mercenary camps they’ve come across, so we seek them out.\n\nWith their help, we defeat the undead. One by one, all the ships are scorch ...
+ wc_arthas_story_65_DESC_group:0 "The captain order me to join the others cutting through the woods. As I picked up an axe, I noticed [arthas.GetTitledFirstName] in a rush out of the camp. Nevermind. I must join the others.\n\nFinally, we break through to the shores, but what we witness dismays us; the ships are burning. [arthas.GetTitledFirstName], our leader, approaches us from the shores, it s ...
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_35_DESC_purge_silver_hand:0 "Some men disobey [arthas.GetHerHis] command and pack their things, they cannot watch this happen. Many paladins of the Silver Hand refuse vehemently."
+ wc_arthas_story_35_DESC_purge_no_silver_hand:0 "[arthas.GetHerHis|U] command splinters our group, the Holier men refuse vehemently. The remaining men follow [arthas.GetHerHis] orders with twisted ease."
+ wc_arthas_story_35_DESC_purge_ending:0 "As for my decision…"
+ wc_arthas_story_35_DESC_evacuate:0 "[arthas.GetTitledFirstName] has made a decision; this city is to be evacuated.\n\n[arthas.GetSheHe|U] will lead a small force inside the city to deliver this news. It seems today will not bring bloodshed, but I sense this plague will spread even further outside of these walls. I cannot help but wonder, what terrible fate awaited these people if they had to sta ...
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_35_TITLE_evacuate:0 "Evacuation of [stratholme.GetNameNoTier]"
+ wc_arthas_story_35_TITLE_quarantine:0 "Quarantine of [stratholme.GetNameNoTier]"
+ 
+ wc_arthas_story_35_DESC_purge_beggining:0 "[arthas.GetTitledFirstNamePossessive] decision is final. Many in our group are in disbelief. [arthas.GetHerHis|U] orders are to purge the entire city… Did we march all the way here to kill our own innocent kind?\n\n[arthas.GetSheHe|U] speaks to us all: "Those of you who have the will to save this land, follow me! The rest of you... get out of my sight!" ...
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_20_DESC:0 "As we approach [andorhal.GetNameNoTier], a slight chill sets as well as an uneasy feeling. I can see it in the eyes of my men as we walk amongst the deserted streets, devoid of any life and littered with boxes of foul grain.\n\nDeeper in the cities, stands a sickly man yet one oozing with power, decaying corpses animated by an unholy force standing guard around [kelthu ...
+ wc_arthas_story_20_OPT_A:0 "Foul knave!"
+ 
+ wc_arthas_story_25_DESC:0 "Our group arrives in [hearthglen.GetNameNoTier] after a long period of marching. The necromancer's words linger in the back in my mind. As we traverse farther into the city, I make note of the soldiers training in the village square.\n\nA footman greets our group: "[arthas.GetTitledFirstName]! During the night, a vast army of undead warriors emerged and began attacking ...
```
done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec)

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,136 @@
+ wc_arthas_story_9_1_OPT_B:0 "[arthas.GetFirstName|U] don't dare to go there!"
+
+ wc_arthas_story_10_TITLE:0 "Outbreak Expedition"
+ wc_arthas_story_10_DESC_beggining:0 "[terenas.GetTitledFirstName], my [ROOT.Char.Custom2('RelationToMeShort', terenas.)], calls upon me to investigate the rumours of a plague in the northlands. My entire upbringing has prepared me for this moment. It is time to fulfill my duty and protect those in need of help, whatever it takes."
```
rumors is done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec), event name left like this because it's the same in the code.

arithon commented:
> * Playing as the random generic Vassal. These events are written as Arthas PoV and are incompatible for generic Vassal player:

At first, I thought this to be an issue as well, but since you're a vassal owning land (and troops) every time, I think it's not that bad. Even a count can be spoken to as mylord and can order his men to take positions, or think about the consequences for his accompanying soldiers.

CrookDoodle reviewed (approved):
All seem to check out.

zumbak04 commented:
> I do not know the exact duration the event last long for lore wise in regards to Arthas going to Northrend and returning back home to slay Terenes, however, I wanted to test to see how far I could get my fleet (originating in Strathholme) to Northrend and back... I conducted this experiment the second Arthas swears to follow Mal'ganis to ends of the Earth. While I understand the variation between to games naval speed and lore-wise, my fleet was only halfway to Northrend for the first time when Arthas was already plunging his blade into his father. I feel like the events should be spaced a bit more realistically at least for the standards of the game. Just a suggestion

It said 1 month and we stick to the lore timelines.

> Whenever you play as anyone other then Arthas and he usurps Lorderon, he marries some random naked woman lmfao, first time it was Jania which made me laugh, but I doubt Jania would want to marry an Undead Arthas and agree to be naked 24/7, even if Arthas dreamed about it... This only happens if you are a vassal or Terenas. Arthas becomes married to the said naked woman the SECOND he become high king (ai arthas surrendered and that is why he is a duke there in the first pic)

Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/822.

> Next, I noticed that sometimes that it isn't Murdain in Northrend but some random other dwarf, is that intended sometimes??

Yes, if Muradin is busy.

zumbak04 commented:
> There should be a special panel that indicates where I could open the Outbreak Expedition page; something like a Great Holy War badge that is obvious to the player. I'm clueless on how I could open the Outbreak Expedition panel. The Icon indicator on top of the expedition's current location didn't show up 99% of the time and is not obvious enough for players.

I use the vanilla interface thus it's up to Paradox to change it.

> Playing as the random generic Vassal. These events are written as Arthas PoV and are incompatible for generic Vassal player:

It's written for both Arthas and vassal, Clarify what exactly you want to change.

zumbak04 commented:
> AI Arthas surrenders, however, he only obtains 2 counties and 2300 levies. Should he be a bit more beefy as this man still has to invade the elves (at least for AI) possibly a special host such as William in the vanilla game?

No, the Scourge is planned to wage these wars, Arthas is a commander.

zumbak04 commented:
> Can you remove Arthas from the outbreak expedition then, the result right now is the expedition is saying Arthas is present in Lordaeron and the events are saying Arthas is in Northrend

No, the activity is a central object of the concept.

TheDrPibb1999 reviewed (approved):
All questions answered, LGTM!

zumbak04 commented:
> Yep I surrendered as well it worked, could you make it so the player is forced to surrender or does Arthas have enough free will at that moment to chose?

Done in 7c3fbd8657830681c078ba520d8fad3ead251c19.

zumbak04 commented:
> * This option should be clarified further on how it would impact the expedition. "To the ends of the earth!" should have something along the line: you pursue Mal'Ganis in the frigid land of Northrend.

Done in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/55945447f6fbd22b66789f51dfdb67294b197f72.

> * If Terenas died during the event, Arthas would get an event to recall him back:

Fixed in 98ecfb20d42b3804b433296d8cb3d926f247adfc.

zumbak04 commented:
> Lastly, I also noticed that is a random chance if Mal'ganis is either wearing armor or normal attire when getting slain by Arthas

Changed in 3c221d3c5891d9f81a1d0fdc6de9908dbf24bf57.

Gaben1025 commented:
> > Can you remove Arthas from the outbreak expedition then, the result right now is the expedition is saying Arthas is present in Lordaeron and the events are saying Arthas is in Northrend
> 
> No, the activity is a central object of the concept.

So shouldn't the expedition follow him to Northend? This looks a little buggy when he is in 2 places at once.

zumbak04 commented:
> So shouldn't the expedition follow him to Northend? This looks a little buggy when he is in 2 places at once.

We can't move the activity. I would like to, but it's impossible as I know..

TheGroxEmpire commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -0,0 +1,140 @@
+ wc_arthas_story_30_DESC_beggining:0 "The weary road draws to an end. In front of us lies"
+ wc_arthas_story_30_DESC_port_city:0 "the port city of [stratholme.GetNameNoTier], home of the Order of the Silver Hand."
+ wc_arthas_story_30_DESC_city:0 "the city of [stratholme.GetNameNoTier]."
+ wc_arthas_story_30_DESC_ending:0 "My scouts return from its gates with wretched news.\n\nThese people have all been infected. They may look fine now, but it is just a matter of time before they turn into the undead!\n\nMy men take positions outside the walls and await my orders. I know I will find [malganis.GetFirstName] inside. But this plague will bring destruction to the rest of my people. My ...
```
The previous possessive pronouns indicates Arthas PoV. Changed it so it is compatible with other expedition members also.
```suggestion
 wc_arthas_story_30_DESC_ending:0 "My scouts return from its gates with wretched news.\n\nThese people have all been infected. They may look fine now, but it is just a matter of time before they turn into the undead!\n\nMy men take positions outside the walls and await my orders. I know I will find [malganis.GetFirstName] inside. But this plague will bring destruction to the rest of our people. Our actions will have dire consequences, not only for the citizens but our followers as well..."
```

arithon replied:
As I wrote as an answer to your previous comment, I thought so at first, too, but since you're a vassal owning land (and troops) every time, I think it's not that bad. Even a count can be spoken to as mylord and can order his men to take positions, or think about the consequences for his accompanying soldiers.

zumbak04 replied:
> As I wrote as an answer to your previous comment, I thought so at first, too, but since you're a vassal owning land (and troops) every time, I think it's not that bad. Even a count can be spoken to as mylord and can order his men to take positions, or think about the consequences for his accompanying soldiers.

I agree.

TheGroxEmpire replied:
Yeah, but this case is the other event. I left the m'lord untouched.

> But this plague will bring destruction to the rest of my people. My actions will have dire consequences, not only for the citizens but my followers as well..."
 
These "my" I think should be replaced with "our" as the vassal doesn't make the choice, Arthas does.

TheGroxEmpire reviewed (commented):
If Terenas died during the event, this happened on the last chain:
<details>

![image](https://user-images.githubusercontent.com/47816228/149820052-48a1face-a8a3-458b-a425-4b516f4de06c.png)
</details>

zumbak04 commented:
> If Terenas died during the event, this happened on the last chain:

During what event?

TheGroxEmpire commented:
> > If Terenas died during the event, this happened on the last chain:
> 
> During what event?

I just killed him after the expedition starts.

Gaben1025 reviewed (approved):
I would add issue for future to move the activity if it becomes possible, the rest (that hasn't been already commented on) looks good to me

zumbak04 commented:
> If Terenas died during the event, this happened on the last chain:

Fixed in d8a284855006429d9f244d28d6ee81f2f0157134.

TheGroxEmpire reviewed (approved):
Looks like my issues are fixed. The locs and the rest look good to me also.

TheDrPibb1999 reviewed (approved):
LGTM

arithon commented on localization/english/event_localization/wc_arthas_expedition_events_l_english.yml:
```diff
@@ -86,12 +86,12 @@
- wc_arthas_story_55_DESC_group_ending:0 "[muradin.GetSheHe|U] says that [muradin.GetHerHis] purpose here is to recover a runeblade called Frostmourne.\n\nWe joined up with [muradin.GetFirstNamePossessive] encampment and are now prepared to battle the undead that roam these frozen lands. Some of [muradin.GetFirstNamePossessive] forces have been cut off from [muradin.GetFirstName] camp. Our expedit ...
+ wc_arthas_story_55_DESC_group_beggining:0 "After a month of sailing, we have arrived in Northrend. The howling wind cuts to the bone.\n\nWe didn't travel far inland until we found a [muradin.GetCulture.GetGroup.GetName] camp and their leader"
+ wc_arthas_story_55_DESC_group_muradin:0 "[muradin.GetFirstName] [muradin.GetDynastyHouseName], [arthas.Custom2('GetTitledFirstNamePossessiveOrMy', ROOT.Char)] old teacher and friend."
+ wc_arthas_story_55_DESC_group_ending:0 "[muradin.GetSheHe|U] says that [muradin.GetHerHis] purpose here is to recover a runeblade called Frostmourne.\n\nWe joined up with [muradin.GetFirstNamePossessive] encampment and are now prepared to battle the undead that roam these frozen lands. Some of [muradin.Custom2('GetFirstNamePossessiveOrMy', ROOT.Char)] forces have been cut off from [muradin.Custo ...
```
This change doesn't make sense, why would some of the forces of an expedition member be cut off?

zumbak04 replied:
> This change doesn't make sense, why would some of the forces of an expedition member be cut off?

Muradin? He had an dwarven expedition looking for Frostmourne in WC3.

arithon replied:
But why have it "GetFirstNamePossessiveOrMy" then? My would target the group member, wouldn't it? So a group member would read "Some of my forces have been cut off my camp.", right?

zumbak04 replied:
> But why have it "GetFirstNamePossessiveOrMy" then? My would target the group member, wouldn't it? So a group member would read "Some of my forces have been cut off my camp.", right?

No, it should be...

> Some of his forces

arithon replied:
As oh, if that's how it's defined, i have no objections.

arithon reviewed (commented):
lgtm, except the one comment

Panabana commented on localization/english/wc_draenor_l_english.yml:
```diff
@@ -12,6 +12,6 @@
  activity_beyond_dark_portal:0 "Invasion of Draenor"
- activity_beyond_dark_portal_name:0 "[ACTIVITY.GetOwner.GetNamePossessiveOrMy|U] Invasion of Draenor"
+ activity_beyond_dark_portal_name:1 "[ACTIVITY.GetOwner.GetShortUINamePossessive|U] Invasion of Draenor"
  activity_beyond_dark_portal_desc:0 "Every brave men and women has been invited to end with the Dark Portal once and for all!"
```
```suggestion
 activity_beyond_dark_portal_desc:0 "Every brave man and woman has been invited to end the Dark Portal once and for all!"
```

arithon replied:
That's nothing that was altered in this PR however...

zumbak04 replied:
I'll transfer this suggestion to the localization branch. If you noticed a typo not related to the PR, create a [Issue](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/new/choose).

zumbak04 replied:
Done in 774798c65451a8b5de80033b5169907f1e245c3e.
